core/api/core.api

```diff
@@ -10193,17 +10652,202 @@ public final class dev/kord/core/entity/interaction/response/PublicMessageIntera
 	public fun withStrategy (Ldev/kord/core/supplier/EntitySupplyStrategy;)Ldev/kord/core/entity/interaction/response/PublicMessageInteractionResponse;
 }
 
-public abstract interface class dev/kord/core/event/Event {
+public abstract interface class dev/kord/core/event/Event : dev/kord/core/KordObject {
 	public abstract fun getCustomContext ()Ljava/lang/Object;
 	public abstract fun getGateway ()Ldev/kord/gateway/Gateway;
-	public abstract fun getKord ()Ldev/kord/core/Kord;
 	public abstract fun getShard ()I
 }
```

Comment:
This is a binary compatible change, `Event` doesn't lose or get new members (`getKord()` is just inherited now).